graphql_compiler/cost_estimation/filter_selectivity_utils.py

```diff
@@ -436,8 +441,13 @@ def get_selectivity_of_filters_at_vertex(schema_info, filter_infos, parameters,
             if filter_info.op_name == "=":
                 # TODO(bojanserafimov): Check if the filter value is in the interval selected
                 #                       by the inequality filters.
-                selectivity = _estimate_filter_selectivity_of_equality(
-                    schema_info, location_name, filter_info.fields
+                collection = None
```

Comment:
This code seems almost identical to the code for `in_collection` filters right above. Is there something conceptually different about the way we compute selectivity for the two types of filters, or would it be easy to merge these two blocks of code into one?

(If the former, we can still combine the two loops through `filters_on_field` because they loop through the same thing)

Reply:
Also, unrelated but I can't comment on the function declaration itself on [line 345](https://github.com/kensho-technologies/graphql-compiler/pull/864/files?file-filters%5B%5D=.py#diff-5dfbb159b89d2baa7f0e0dd13732e415R345) but let's move the type hints from the docstring to the actual code since we're modifying this function.